src/hooks/use-row-selection/use-row-selection.spec.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { render } from '../../test-utils';
```

Comment:
Any reason to not use [@testing-library/react-hooks](https://github.com/testing-library/react-hooks-testing-library)?

Reply:
Hmm, not really. However, I do like the idea of testing this particular hook with a layout component, providing an example of how to use it in practice.